src/StmtEnums.h

```diff
@@ -32,6 +32,7 @@ enum StmtTag {
     STMT_NULL,
     STMT_ASSERT,
     STMT_EXTERN, // for custom Stmt subclasses provided by plugins
+    STMT_NATIVE, // statements that call a std::function from c++
```

Comment:
Seeing STMT_CPP above, maybe that could be re-used. Though probably better to have a new one. I start to like your earlier and more specific `STMT_STD_FUNCTION` more than the opaque `STMT_NATIVE`.. sorry ;-)

Reply:
`STMT_CPP` is for script-opt, so it probably isn't appropriate here.

Reply:
I renamed it back to STMT_STD_FUNCTION and also fixed the array of strings that I missed before.